grails-app/taglib/aatams/GeometryTagLib.groovy

```diff
@@ -25,9 +26,13 @@ class GeometryTagLib  {
             if (attrs.value == null) {
             }
             else {
-                lon = attrs.value?.getCoordinate()?.x
-                lat = attrs.value?.getCoordinate()?.y
-                srid = attrs.value?.getSRID()
+
+                def value = attrs.scramble && attrs.scramble == "true" ?
+                    GeometryUtils.scrambleLocation(attrs.value) : attrs.value
```

Comment:
I checked this again, because I couldn't remember - it seems to be fine.

The code delegates to https://github.com/aodn/aatams/blob/master/grails-app/utils/au/org/emii/aatams/util/GeometryUtils.groovy where the user authentication status is checked.